src/v2/view-builder/internals/BaseModel.js

```diff
@@ -11,6 +11,13 @@
  */
 import { _, Model } from 'okta';
 
+const FORM_NAME_TO_OPERATION_MAP = {
+  'select-authenticator-unlock-account': 'UNLOCK_ACCOUNT',
+  'identify': 'PRIMARY_AUTH',
+  'select-identify': 'PRIMARY_AUTH',
```

Comment:
Doesn't find what view represented by `select-identify`?
Looks like it's just an action to switch from register page to identify page?

Reply:
enroll-profile-new.json and enroll-profile.json have an identifier form value for it:
```
{
        "rel": [
          "create-form"
        ],
        "name": "select-identify",
        "href": "http://localhost:3000/idp/idx",
        "method": "POST",
        "accepts": "application/vnd.okta.v1+json",
        "value": [
          {
            "name": "identifier",
            "label": "identifier"
          },
          {
            "name": "stateHandle",
            "value": "01r2p5S9qaAjESMFuPzt7r3ZMcZZQ_vvS0Tzg56ajB",
            "visible": false
          }
        ]
      }

Reply:
I didn't find a View represented for `select-identify` in ViewFactory hence the ask.

I may miss something or we need to double check if the mock is out of dated.